passport-auth.js

```diff
@@ -0,0 +1,20 @@
+var passport = require('passport');
+const GoogleStrategy = require('passport-google-oauth20').Strategy;
+
+passport.use(new GoogleStrategy({
+    clientID: process.env.CLIENT_ID,
+    clientSecret: process.env.CLIENT_SECRET,
```

Comment:
Let's rename `CLIENT_ID` to `PASSPORT_CLIENT_ID`, and `CLIENT_SECRET` to `PASSPORT_CLIENT_SECRET`. This is because we want to make environmental variable naming specific to its use case.